__main__.py

```diff
@@ -8,16 +8,19 @@
 NLP Application for Summarizing Finance Article
  """)
 
-ticker = st.text_input('Please enter a stock ticker that you would like to learn more about:')
+# add the drop down menu
+sp500_df = pd.read_csv('constituents.csv')
+sp500_ls = sp500_df['Symbol'].tolist()
+ticker = st.selectbox('Please enter a stock ticker that you would like to learn more about:', sp500_ls)
+
+# ticker = st.text_input('Please enter a stock ticker that you would like to learn more about:')
 
 st.write("""1. """, ticker)
```

Comment:
Is it possible to get phrases through a mock function, I believe it will be much easier for future implementation. If you plan to do so in this PR, I recommend to use a list of phrase as return value. 

ex: 
["""summary1""", """summary2""", ...]

Reply:
Thanks for reviewing. I am not sure if I understand it correctly...
This PR is for providing choices of tickers for users. 
In Viet's branch, phrases are generated from `phrases_output.txt`.
So after merging, return value can come out via a for loop in his branch, starting from line 25

Reply:
That's true, it's my fault!!